core/templates/services/context.service.spec.ts

```diff
@@ -160,6 +181,17 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('topic_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(true);
+      });
+
+    it('should affirm the page context other than exploration editor ' +
+      'and player', ()=> {
```

Comment:
This seems a bit confusing. Maybe change it to 'should not report exploration context when the context is not editor or player related'?

Reply:
Done!